scripts/upgrade

```diff
@@ -1,35 +1,47 @@
 #!/bin/bash
 
+#=================================================
+# GENERIC START
 #=================================================
 # IMPORT GENERIC HELPERS
 #=================================================
 
-# Source YunoHost helpers
 source _common.sh
 source /usr/share/yunohost/helpers
 
-#=================================================
-# MANAGE SCRIPT FAILURE
-#=================================================
-
-ynh_abort_if_errors
-
 #=================================================
 # LOAD SETTINGS
 #=================================================
+ynh_script_progression --message="Loading installation settings..." --time --weight=1
 
-# Retrieve app settings
 app=$YNH_APP_INSTANCE_NAME
-domain=$(ynh_app_setting_get "$app" domain)
-
-
-dockerapp_ynh_loadvariables
+domain=$(ynh_app_setting_get --app=$app --key=domain)
+path_url="/"
+is_public=$(ynh_app_setting_get --app=$app --key=is_public)
+port=$(ynh_app_setting_get --app=$app --key=port)
+data_path=/home/yunohost.app/docker/$app
```

Comment:
Idk how much freedom/constrains we have for this but ideally I'd try to stick to ''/home/yunohost.$app/" or "/home/yunohost.app/$app" to have sort of a standard compared to other app (this could help for future stuff ...)

Reply:
I think it's because Portainer is itself a Docker container. It will have its location under the docker folder, along with the other containers.
So it is better to have /home/yunohost.app/docker/portainer ,  /home/yunohost.app/docker/some_other_docker_container.
It will be easy to remove all the container through remove script in future, if its required.